framer/Components/DeviceComponent.coffee

```diff
@@ -73,6 +73,8 @@ class exports.DeviceComponent extends BaseClass
 		if @_setupDone
 			return
 
+		@_buildInDevices = _.keys Devices
```

Comment:
Also, in theory it's a waste to set this for every instance. You should make it a static global in the module.